.github/workflows/adhoc_build.yml

```diff
@@ -34,11 +34,13 @@ jobs:
         with:
           python-version: "3.11"
           architecture: x64 # Otherwise the runner will try to download Python arm64, which is not available. x64 has support for both archs (universal2).
-      - name: Install latest stable
-        uses: actions-rs/toolchain@v1
+      - name: Install latest stable Rust toolchain
+        uses: actions-rust-lang/setup-rust-toolchain@v1
         with:
           toolchain: stable
           override: true
+          cache: false
+          rustflags: "-A warnings"
```

Comment:
One of the deps of our rust library has a few warnings, and by default, this action will treat warnings as errors and compilation of `fig_kiwi` will fail. So I'm mimicking the behaviour of the previous action here.